src/components/Content.js

```diff
@@ -20,7 +21,14 @@ export default class Content extends Component {
             {this.props.viewType}
           </Button>
         </div>
-        <p>{explain}</p>
+        {
+          isArray(explain) 
```

Comment:
Why not use `Array.isArray` here directly? I'm not familliar with jsx.

Reply:
I want write a collection of javascript utility function. because, later, We will have a lot of reuse logic.

Reply:
Also, `isArray` will more be more complex, not like now